witness/golang/cmd/witness/internal/witness/witness.go

```diff
@@ -301,3 +294,25 @@ func (w *Witness) setChkptData(tx *sql.Tx, logID string, c []byte, rng []byte) e
 		logID, c, rng)
 	return tx.Commit()
 }
+
+// setInitChkptData stores the data for an initial checkpoint and, if using one,
+// its associated compact range.
+func (w *Witness) setInitChkptData(tx *sql.Tx, logID string, c *log.Checkpoint, cRaw []byte, rngRaw [][]byte) error {
```

Comment:
The refactoring of this method would be better in a different PR. It took me a while to realize we weren't making changes to the handling of compact ranges when I saw this deleted in the other place.

Reply:
Yup, agreed - sorry for the tangled mess!  This is what happens apparently when I am going on holidays and try to rush things :/.